src/target/riscv/riscv.c

```diff
@@ -4565,12 +4570,8 @@ static void riscv_invalidate_register_cache(struct target *target)
 	if (!target->reg_cache)
 		return;
 
-	LOG_DEBUG("[%d]", target->coreid);
+	LOG_TARGET_DEBUG(target, "Invalidating register cache of hart: [%d]", target->coreid);
```

Comment:
The target gets us better info than the coreid number.
```suggestion
	LOG_TARGET_DEBUG(target, "Invalidating register cache");
```